agenta-cli/agenta/templates/sentiment_analysis/app.py

```diff
@@ -0,0 +1,40 @@
+import agenta as ag
+from langchain.chat_models import ChatOpenAI
+from langchain.output_parsers import StructuredOutputParser, ResponseSchema
+from langchain.prompts import ChatPromptTemplate, HumanMessagePromptTemplate
+
+default_prompt = "Categorise the sentiment of this text: {text}"
+
+
+@ag.post
+def generate(
+    text: str,
+    temperature: ag.FloatParam = 0.9,
+    prompt_template: ag.TextParam = default_prompt,
+) -> str:
+    llm = ChatOpenAI(temperature=temperature)
+
+    response_schemas = [
+        ResponseSchema(name="category", description="name of sentiment category"),
+        ResponseSchema(
+            name="degree",
+            description="percentage of how much this sentiment is predicted to be of in this category",
```

Comment:
These should be part of the parameters of the apps, otherwise the user would not be able to experiment on the prompts.

Reply:
@mmabrouk please check again